components/auth/login.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="stylesheet" href="/style.css" />
+    <title>AdornBox</title>
+  </head>
+  <body>
+    <div class="main-container">
+      <main class="home-container">
+        <div id="main-navbar" class="ab-navbar-shadow"></div>
+        <div class="ab-auth-wrapper">
+          <div class="card ab-auth-card">
+            <div class="card-title">Login</div>
+            <div
+              class="card-text ab-auth-card-body ab-display-flex ab-flex-flow-column"
+            >
+              <div class="input-group">
+                <label>Username/Email *</label>
+                <input
+                  type="email"
+                  class="au-input"
```

Comment:
Use self-declarative naming convention instead of shorthands.

Reply:
AU is for reference of my component library AdornUI which I've used here as helper class.
I will add proper comments & docs for same.